source/administration/console/subnet-registration.rst

```diff
@@ -14,17 +14,19 @@ SUBNET Registration and Support
 You can use the MinIO Console to perform several of the license and subscription related functions available in MinIO, such as:
 
 - View the license you are currently using for your MinIO deployment.
-- Subscribe to |SUBNET|.
-- Manage the deployment's SUBNET license.
+- Subscribe to a commercial license, which includes access to |SUBNET|.
+- Manage the deployment's Enterprise license.
 - Access Support tools for sharing with MinIO Engineering.
+- Review differences between license options.
 
 License
 -------
 
-MinIO offers two licensing options: 
+MinIO offers three licensing options: 
 
 #. Open source with the :minio-git:`GNU AGPLv3 license <mc/blob/master/LICENSE>`
-#. Paid commercial license with included support direct from MinIO Engineers
+#. Enterprise Lite, a `commercial license <https://min.io/pricing?ref=docs>`__ with included support direct from MinIO Engineers
+#. Enterprise Plus, a `commercial license <https://min.io/pricing?ref=docs>`__ with included support direct from MinIO Engineers, longer release cycle, shorter SLA, and other benefits
 
 This page shows the current license status of the deployment.
```

Comment:
While we are in here, might as well clarify what `This` refers to. I don't remember exactly how that page is titled, perhaps:
```suggestion
The :guilabel:`License` page shows the current license status of the deployment.
```